nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -144,6 +138,23 @@ scenarioTypes {
     }
     category: "Category2"
   }
+  "batch-dev": {
+    deploymentConfig: ${flinkDeploymentConfig}
+    modelConfig: {
+      classPath: ["model/devModel.jar", "model/flinkExecutor.jar", "components/flink/flinkBase.jar", "components/flink-dev/flinkTable.jar", "flink-dropwizard-metrics-deps/"]
+      rocksDB: {
+        enable: true
+        enable: ${?FLINK_ROCKSDB_ENABLE}
+      }
+      components.flinkTable {
+        // The table configuration file should be mounted on the Flink Taskmanager container under this path. You can
+        // override this path through the 'TABLES_DEFINITION_FILE' env variable for a given Flink service.
+        tableDefinitionFilePath: "../../../nussknacker-dist/src/universal/conf/tables-definition.sql"
```

Comment:
In which situation this default is used? I know that I recommended to add some default but I see know that this value is pretty hacky. Maybe we should have some dumb default and more explicitly pass correct values in each cases?